src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
+        Ball randomBall = new Ball();
+
+        randomBall.setNumber(new Random().nextInt(100));
+        randomBall.setColor(supplier.getRandomColor());
+
+        return randomBall;
```

Comment:
same lets make Random and ColorSupplier as class level variables, not to create new instances every time we call getRandomBall() method. Lets move number 100 to constant https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names, and remove all redundant empty lines